src/components/layouts/dropzone/Dropzone.scss

```diff
@@ -0,0 +1,41 @@
+@import "../../variables";
+
+.dropzone {
+  width: 100%;
+  transition: $linear;
+  padding: 30px 0;
+
+  display: flex;
+  align-items: start;
```

Comment:
I see a warning regarding this:
```
start value has mixed support, consider using flex-start instead
```
![image](https://user-images.githubusercontent.com/5430498/78799927-7c58cf00-79c3-11ea-9112-349df7b0c478.png)